apps/desktop/src/lib/vbranches/ownership.ts

```diff
@@ -203,3 +205,62 @@ export class SelectedOwnership {
 		return this.selection.size === 0;
 	}
 }
+
+interface OwnershipUpdateParams {
+	selectedFileIds: string[];
+	files: AnyFile[];
+	selectedOwnership: Writable<SelectedOwnership> | undefined;
+}
+
+export function updateOwnership(params: OwnershipUpdateParams) {
+	const selectedFiles = params.selectedFileIds
+		.map((id) => unstringifyFileKey(id))
+		.map((id) => params.files.find((f) => f.id === id))
+		.filter((f): f is AnyFile => !!f);
+
+	if (selectedFiles.length === 0) return;
+
+	// Only one file selected
+	//
+	// Select all hunks if none are selected,
+	// otherwise ignore all hunks
```

Comment:
Any chance you could format and punctuate this a bit better? I see one comma, and two questionable line breaks 😂

Reply:
Don't, you, like

.them?

Reply:
Haha, I'll just remove them 👍